d2/src/main/java/com/linkedin/d2/xds/LoadBalanceTaskPool/NewLoadBalancerTaskthreadPool.java

```diff
@@ -0,0 +1,29 @@
+package com.linkedin.d2.xds.LoadBalanceTaskPool;
+
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.RejectedExecutionHandler;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+
+public class NewLoadBalancerTaskthreadPool {
+  private ThreadPoolExecutor _threadPoolExecutor;
+  private int _corePoolSize = 2;
+  private int _maximumPoolSize = 3;
+  private long _keepAliveTime = 200;
+  private int _queueSize = 1000;
```

Comment:
It would be better if these parameters could be managed through cfg2, possibly from the global configuration, allowing services to customize them or use default values if not specified. Once the migration from the oldLB to the newLB is complete, we can clean up all of these configurations, including the threadpool.

Reply:
Thanks. That make sense.

What I have done :
- use `D2ClientConfig` , which could be wired with `container`
- If there is no `ThreadPool` setted by `container`, the `DualReadLoadBalancer` will create a default `ThreadPool`